sbol_utilities/conversion.py

```diff
@@ -474,6 +490,8 @@ def genbank2sbol():
                         help='Print running explanation of conversion process')
     parser.add_argument('--allow-genbank-online', dest='allow_genbank_online', action='store_true', default=False,
                         help='Perform GenBank conversion using online converter')
+    parser.add_argument('--force-new-converter', dest='force_new_converter', action='store_true', default=False,
```

Comment:
You also need to add this argument for `def main()` as well

Reply:
Also added it to the `sbol2genbank()` function.